ui/src/pages/Dashboard/Dashboard/Overview.vue

```diff
@@ -442,6 +472,179 @@ export default {
         this.tableData2.data.push({ title: `3. ${ent[2].name} - ${ent[2].total} Nodes` });
         this.tableData2.data.push({ title: `4. ${ent[3].name} - ${ent[3].total} Nodes` });
         this.tableData2.data.push({ title: `5. ${ent[4].name} - ${ent[4].total} Nodes` });
+
+        this.paymentAddress = new Map();
```

Comment:
Looks pretty complicated, try to simplify

Reply:
Got it